spring-boot/dependency-injection/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you please add the Maven Wrapper to this module so that it runs on any machine?